lib/sdf-server/src/service/v2/view/remove_view.rs

```diff
@@ -28,6 +28,11 @@ pub async fn remove_view(
 
     let force_change_set_id = ChangeSet::force_new(&mut ctx).await?;
 
+    let current_view_list = View::list(&ctx).await?;
+    if current_view_list.len() == 1 {
```

Comment:
This is a reasonable thing by itself, but don't we want to actually prevent users from deleting the default view? Like, ```
if (view_id == View::get_id_for_default(ctx)) {
  return Err(ViewError::CantDeleteDefaultView);
}
```

It seems like even if there are multiple views, we don't want the default view deleted.

Reply:
We do not want to restrict that - we are going to default to another view if there is one